utils/tests.py

```
@@ -276,3 +277,114 @@ def test_output_path_parsing_failure(self, mock_subprocess_run):
+    def test_normalize_name_removes_narrated_by(self):
+        """Verify 'narrated by' and narrator names are removed."""
+        result = self.tool.normalize_name("Great Book narrated by Jane Doe")
+        self.assertNotIn("narrated by", result.lower())
+        self.assertNotIn("jane doe", result.lower())
+        self.assertIn("great", result.lower())
+        self.assertIn("book", result.lower())
+
+    def test_normalize_name_removes_standalone_years(self):
+        """Verify standalone years are removed but other text preserved."""
+        result = self.tool.normalize_name("Book Title 2020 Edition")
+        self.assertNotIn("2020", result)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertNotIn`

Replace `assertNotIn(...)` with `assert ...`

(PT009)